contrib/babelfishpg_tsql/src/pltsql.h

```diff
@@ -2348,4 +2348,29 @@ extern bool validate_special_function(char *proc_nsname, char *proc_name, int na
  */
 extern char *tsql_format_type_extended(Oid type_oid, int32 typemod, bits16 flags);
 
+/* Restricted objects definition */
+typedef struct
+{
+	const char *name;
+	ObjectType type;
+} RestrictedObject;
+
+/* Separate arrays for each schema */
+static const RestrictedObject master_dbo_objects[] = {
+	{"xp_qv",					OBJECT_PROCEDURE},
+	{"xp_instance_regread",		OBJECT_PROCEDURE},
+	{"sp_addlinkedserver",		OBJECT_PROCEDURE},
+	{"sp_addlinkedsrvlogin",	OBJECT_PROCEDURE},
+	{"sp_droplinkedsrvlogin",	OBJECT_PROCEDURE},
+	{"sp_dropserver",			OBJECT_PROCEDURE},
+	{"sp_enum_oledb_providers",	OBJECT_PROCEDURE},
+	{"sp_testlinkedserver",		OBJECT_PROCEDURE}
+};
+
+static const RestrictedObject msdb_dbo_objects[] = {
+	{"syspolicy_configuration",				OBJECT_VIEW},
```

Comment:
We are also creating `dbo.sysdatabases` view as per of database create, I think we should restrict that as well.

Reply:
This is already restricted: 
```
1> drop view sys.sysdatabases
2> go
Msg 33557097, Level 16, State 1, Server BABELFISH, Line 1
must be owner of view sysdatabases
1> 
```

Reply:
Please check it from PSQL endpoint as well. We need to handle that also.
```
babelfish_db=> drop view db3_dbo.sysdatabases;
DROP VIEW
```

Reply:
Please address this.